autogpt_platform/frontend/src/services/feature-flags/use-get-flag.ts

```diff
@@ -37,7 +37,7 @@ export type FlagValues = {
 const isPwMockEnabled = process.env.NEXT_PUBLIC_PW_TEST === "true";
 
 const mockFlags = {
-  [Flag.BETA_BLOCKS]: [],
+  [Flag.BETA_BLOCKS]: ["8b2a7b3c-6e9d-4a5f-8c1b-2e3f4a5b6c7d"], // Human-in-the-Loop block
```

Comment:
We probably don't need this unless we have E2E tests that assert over the new _Humain in the loop_ feature ( _which we don't_ ), so we can remove 👍🏽